spec/models/event_schedule_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe EventSchedule, type: :model do
+  describe '.populate_upcoming_show_and_tell!' do
+    subject { described_class.populate_upcoming_show_and_tell! }
+    before { travel_to Date.new(2017, 1, 1) }
+    after { travel_back }
+
+    let(:fridays) do
```

Comment:
I've taken care of this but this comment won't show up as outdated as the line changed is below the line commented on.